app/helpers/dor_object_helper.rb

```diff
@@ -56,7 +56,7 @@ def has_been_accessioned?(pid)
   end
 
   def last_accessioned_version(pid)
-    Dor::Services::Client.object(pid).sdr.current_version
+    PreservationClient.current_version(pid)
```

Comment:
Should PreservationClient be extracted to a Gem.  This is something dor-services-app could also use.

Reply:
Per discussion after standup this morning:  we'll extract it to a gem after we have a second use case.